plugin/PDFView.qml

```diff
@@ -218,6 +227,15 @@ SilicaFlickable {
         }
     }
 
+    Connections {
+        target: document
+        onSearchModelChanged: {
+            if (document.searchModel) {
+                document.searchModel.onCountChanged.connect(moveOnceToFirstMatch)
```

Comment:
Just thinking as this connect()/disconnect() feels a bit hacky, would this be any better:

```
Connections {
   target: document
   onSearchModelChanged: initialMatch.moveToFirst = true
}
Connections {
   id: initialMatch
   property bool moveToFirst: true
   target: document.searchModel
   onTargetChanged: moveToFirst = true
   onCountChanged: {
      if (document.searchModel.count == 0) {
        moveToFirst = true
      } else if (moveToFirst) {
        moveToSearchMatch(0)
        moveToFirst = false
      }
   }
}
```

Would be agnostic whether model is changed or cleared and shouldn't care about null models.